CMakeLists.txt

```diff
@@ -814,7 +818,7 @@ endif()
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 if(MINGW)
   set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wa,-mbig-obj")
-  set(EXTRA_LIBRARIES mswsock;ws2_32;iphlpapi)
+  set(EXTRA_LIBRARIES mswsock;ws2_32;iphlpapi;bcrypt)
```

Comment:
This is required for successful build of tests of master branch.
It's not required for is_hdd() implementation itself.

[Changed the default random_generator implementation to use ](https://github.com/boostorg/uuid/commit/a8a1ec3d826b04d2c842e79ecd18cff6a991fd29)
boost::uuid::random_generator is being used only in "tests/functional_tests/transactions_flow_test.cpp" but included in few other places.